lucene/CHANGES.txt

```diff
@@ -65,6 +65,8 @@ Other
 
 * LUCENE-8768: Fix Javadocs build in Java 11. (Namgyu Kim)
 
+* LUCENE-8975: Code Cleanup: Use entryset for map iteration wherever possible.
+
```

Comment:
can you move it under `Lucene 8.3.0` instead of `Lucene 9.0.0`, I plan to backport this to 8.x

Reply:
Done